internal/pkg/service/buffer/store/export_test.go

```diff
@@ -265,85 +265,3 @@ config/export/00001000/github/github-issues
 	// Check keys
 	etcdhelper.AssertKVs(t, store.client, ``)
 }
-
-func TestStore_DeleteExportBaseListOp(t *testing.T) {
-	t.Parallel()
-
```

Comment:
Tento test som zmazal.
Nema zmysel testovat iba jednu cast mazania a ostatne nie.
Je to pokryte ostatnymi testami, napr. `close slice`, kde sa nakonci vsetko vymaze.

Testy na ostatne delete operacie chybaju uplne, mozno sa casom doplnia.